js/gdm/authPrompt.js

```diff
@@ -636,7 +659,7 @@ var AuthPrompt = class {
         this.setQuestion(_("Enter unlock code provided by customer support:"));
```

Comment:
While we're changing the other string here, please consider addressing https://phabricator.endlessm.com/T27290. I suggest we change this string to:

> Enter unlock code:

and clarify what this means in the longer string.

Reply:
Done.